### ISSUE TITLE
Big performance gap between released Pythia model and paper?

### ISSUE THREAD

**Issue by gaotongxiao** [facebookresearch/mmf#654]
## ❓ Questions and Help
I downloaded [vqa2 train only](https://dl.fbaipublicfiles.com/pythia/pretrained_models/vqa2/pythia.pth) model and modified `batch_size` in `projects/pythia/configs/vqa2/defaults.yaml` from 512 to 256 and `num_workers` to 0. Then I used the following command to generate the prediction for validation set:
```
mmf_predict dataset=vqa2 model=pythia config=projects/pythia/configs/vqa2/defaults.yaml run_type=val checkpoint.resume_file=data/models/pythia.pth
``` 

I used VQA's official script to evaluate the generated json but only got 41.33% accuracy, which is significantly different from 68.71% that the paper has claimed. 

I know that I'm evaluating on val and the paper evaluated on test, and that the docs say pre-trained Pythia wasn't trained on augmented data. But should such a huge gap be expected or am I doing something wrong?

**Comment by apsdehal** [comment]
I can confirm that this is indeed the case as of now and there seems to be a regression in the master. I will have further look into why this is happening.

**Comment by apsdehal** [comment]
The models that you are using are outdated and use old features, that's why the performance is low. We will upload the new models to the model zoo but that can take time. If it is urgent we suggest that you train the model on your end.
